network/dime_message.ksy

```diff
@@ -0,0 +1,74 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  xref:
+    wikidata: Q1227457
+  license: CC0-1.0
+  bit-endian: be
+  endian: be
+doc: |
+  Direct Internet Message Encapsulation (DIME)
+  is an old Microsoft specification for sending and receiving
+  SOAP messages along with additional attachments,
+  like binary files, XML fragments, and even other
+  SOAP messages, using standard transport protocols like HTTP.
+doc-ref: 
+  - http://xml.coverpages.org/draft-nielsen-dime-02.txt
+  - https://docs.microsoft.com/en-us/archive/msdn-magazine/2002/december/sending-files-attachments-and-soap-messages-via-dime
+seq:
+  - id: records
+    type: record
+    repeat: eos
+types:
+  padding:
+    doc: padding to the next 4-byte boundary
+    seq:
+      - id: boundary_padding
+        size: (- _io.pos) % 4
+  record:
+    doc: each individual fragment of the message
+    seq:
+      - id: version
+        type: b5
+      - id: is_first_record
+        type: b1
+      - id: is_last_record
+        type: b1
+      - id: is_chunck_record
```

Comment:
What does this mean? Is `chunck` an abbreviation of something or just a misspelled `chunk`?

Reply:
It is indeed a misspelled `chunk`